spog/ui/src/pages/search.rs

```diff
@@ -155,6 +155,48 @@ pub fn search(props: &SearchProperties) -> Html {
         },
     );
 
+    let onsort_advisory = {
+        let search_params_state = advisory.search_params.clone();
+        use_callback(
+            move |sort_by: (String, bool), search_params| {
+                if let SearchMode::Simple(simple) = search_params {
+                    let mut simple = simple.clone();
+                    simple.set_sort_by(sort_by);
+                    search_params_state.set(SearchMode::Simple(simple));
+                };
+            },
+            (*advisory.search_params).clone(),
+        )
+    };
+
+    let onsort_sbom = {
+        let search_params_state = sbom.search_params.clone();
+        use_callback(
+            move |sort_by: (String, bool), search_params| {
+                if let SearchMode::Simple(simple) = search_params {
+                    let mut simple = simple.clone();
+                    simple.set_sort_by(sort_by);
+                    search_params_state.set(SearchMode::Simple(simple));
+                };
+            },
+            (*advisory.search_params).clone(),
```

Comment:
I guess this should be `sbom`, not `advisory`.

Reply:
Thanks, you are right. It has been addressed in my last commit